src/hypermedia-constants.js

```diff
@@ -600,6 +606,7 @@ export const Actions = {
 	},
 	workingCopy: {
 		checkout: 'checkout',
-		checkin: 'checkin'
+		checkin: 'checkin',
```

Comment:
Brain fart this should be `commit`.

Reply:
Note: Quiz uses `checkin` so I'll be adding a separate one for `commit`.

Reply:
I wonder if we need to work with them to standardize on a single term (I suspect their `checkin` means the same thing as our `commit`).